schema/encoding/jsonschema/all_go16_test.go

```diff
@@ -0,0 +1,11 @@
+// +build !go1.7
+
+package jsonschema_test
```

Comment:
@yanfali, This is the Go way to test the public interface of your package. It makes you having to import the package you are testing, and only give access to the public interface. Presumably you could have done the same for the tests you put in the encoding package.